README.md

```diff
@@ -18,7 +22,12 @@
 
 ---
 
-**React Native OMH Maps** eases the process to implement interactive maps in React Native apps on both iOS and Android. Under the hood, for Android it uses the Android [OMH Maps Client Library](https://github.com/openmobilehub/android-omh-maps). Supports both Google Mobile Services (GMS) and non-GMS configurations, with a unified interface for easy incorporation of Google Maps, OpenStreetMap, Mapbox, Azure Maps and other supported third-party maps providers.
+**React Native OMH Maps** eases the process to implement interactive maps in React Native apps on both iOS and Android. Under the hood, for Android it uses the Android [OMH Maps Client Library](https://github.com/openmobilehub/android-omh-maps). Supports both Google Mobile Services (GMS) and non-GMS configurations, with a unified interface for easy incorporation of Google Maps, OpenStreetMap, Mapbox, Azure Maps and other supported third-party maps providers. It provides a single API to use different maps providers:
+
+- Google Maps via `@openmobilehub/maps-plugin-googlemaps`
+- OpenStreetMap via `@openmobilehub/maps-plugin-openstreetmap`
+- Mapbox via `@openmobilehub/maps-plugin-mapbox`
+- Azure Maps via `@openmobilehub/maps-plugin-azuremaps`
 
```

Comment:
Should we add?
```
- Apple Maps via `@openmobilehub/maps-core`
```

Reply:
@artus9033 @maciekBudzinski what do you think? Those are plugins, and maps is separate library, I am not sure if we should add it or not

Reply:
Maybe I would add separate paragraph under that says something like:

iOS implementation relies on `react-native-maps` library and supports the following providers:
- ...
- ...
All of the iOS providers are available by installing `@openmobilehub/maps-core`.

WDYT?